sig-contributor-experience/surveys/k8s_survey_analysis/prepare_2018.py

```diff
@@ -25,7 +25,7 @@
     'Most_Important_Proj:_GH_Mgmt':'Most_Important_Proj:_GitHub_Management',
     'Most_Important_Proj:_Contributor_Summits':'Most_Important_Proj:_Delivering_valuable_contributor_summits_at_relevant_events',
     'Most_Important_Proj:_Keeping_community_safe': 'Most_Important_Proj:_Keeping_our_community_safe_on_our_various_communication_platforms_through_moderation_guidelines_and_new_approaches',
-    'Check_for_news:_k-dev_ML':'Check_for_news:_kubernetes-dev@_mailing_list',
+    'Check_for_news:_k-dev_ML':'Check_for_news:_dev@kubernetes.io_mailing_list',
```

Comment:
this file does not need to be changed as its an archive so we should keep the history